src/genie/libs/parser/iosxr/tests/ShowRibTablesSummary/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,12 @@
+expected_output = {
+                'rib_summary': {
```

Comment:
same with above. `rib_summary` key is not mandatory.

Reply:
Thank you, I've removed the "rib_summary" key and updated the schema.